docs/reference/how-to/size-your-shards.asciidoc

```diff
@@ -68,7 +68,17 @@ This decreases the number of segments, which means less metadata is kept in
 heap memory.
 
 Every mapped field also carries some overhead in terms of memory usage and disk
-space. By default {es} will automatically create a mapping for every field in
+space. While the exact amount resource usage of each additional field depends
+on the type of field, an additional 1024B in memory use  for each additional
+mapped field on data nodes is a good approximation in most cases.
+As a result, the more mapped fields you have per index, the fewer
+indices and shards each individual data node will be able to hold,
```

Comment:
Similarly here, could we expose relevant metrics in nodes stats?

Reply:
Yea, as with mapping stats this should be quite straight forward actually

Reply:
Similarly, see #86639